src/core/server/rendering/rendering_service.tsx

```diff
@@ -171,7 +171,7 @@ export class RenderingService {
     };
   }
 
-  public async stop() {}
+  public async stop() { }
```

Comment:
Could you delete the empty space here? it causes a linter error @Alankarsharma

Reply:
done, thanks for the review